### PR TITLE
Fix month names issue for the custom CultureInfo in the `BitDatePicker` component (#2073)

### LINKED ISSUE #2073: Month names issue for the custom CultureInfo in the `BitDatePicker` component
In the `Jalali BitDatePicker` section, the style of some month's names is not correct.

Screenshot 2022-05-23 201300

And, I have a question here, is it ok to use Persian sentence, when all words here are in finglish?

### PR DESCRIPTION
**Pull request by sajadkardel**
this closes #2073

### CODE REVIEW

**Comment by sajadkardel** [comment]
To review the result of these changes, you need to change parts of the project settings. The image below may help you to review the changes -_-

Untitled

(the colorful border is flex box)

**Review by msynk — APPROVED** [review]

**Comment by msynk** [comment]
@sajadkardel 
could you plz add the full month name as a title (tooltip), since in your solution you used the ellipsis cut-off

**Review by msynk — CHANGES_REQUESTED** [review]

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/DatePicker/BitDatePicker.razor [hunk 1]
```diff
@@ -218,6 +218,7 @@
                                                     disabled="@CheckMonthForMaxAndMinDate(monthIndex)"
                                                     type="button"
                                                     aria-label="@Culture?.DateTimeFormat.GetMonthName(monthIndex)"
+                                                    title="@Culture?.DateTimeFormat.GetMonthName(monthIndex)"
```
[the remark this answers is not in the document]
Comment (sajadkardel) [reply src/Client/Web/Bit.Client.Web.BlazorUI/Components/DatePicker/BitDatePicker.razor:221]: also `CheckMonthForMaxAndMinDate` is called 2 time in this section.
change this too or not?
@msynk

[the remark this answers is not in the document]
Comment (msynk) [reply src/Client/Web/Bit.Client.Web.BlazorUI/Components/DatePicker/BitDatePicker.razor:221]: not related to this issue

**Pull request merged**